django_declarative_apis/machinery/filtering.py

```diff
@@ -109,20 +125,55 @@ def _get_filtered_field_value(
 
 # TODO: make this method less complex and remove the `noqa`
 def _apply_filters_to_object(  # noqa: C901
-    inst, filter_def, expand_children=None, klass=None
+    inst,
+    filter_def,
+    expand_children=None,
+    klass=None,
+    filter_cache=None,
+    debug_indent="",
```

Comment:
This is debug-specific code. A slight shift tracking depth would put a spin on it (then the prefix could just be `" " * depth`.